src/VirtoCommerce.Platform.Web/wwwroot/js/common/directives/fallbackSrc.js

```diff
@@ -1,5 +1,6 @@
 ///https://github.com/dcohenb/angular-img-fallback
 ///Angular directives that handles image loading, it has fallback-src to handle errors in image loading and loading-src for placeholder while the image is being loaded.
+// Doesn't work with falsy src values!
```

Comment:
May you a little bit explain that?

Reply:
If a ng-src equals null or empty string or undefined, this directive doesn't replace it on a fallback source.